### DISCUSSION TITLE
UI color

### DISCUSSION THREAD

**Discussion by MatthewJRobertson94** [MatthewJRobertson94/Tafe230132#2] [General]
What colour should the UI be? Any suggestions?

**Comment by vincentx1908** [comment]
I think Azure Hex: #F0FFFF RGB (240, 255, 255) is a popular colour for background! :)

**Comment by MatthewJRobertson94** [comment]
Are you free anytime tonight or during the day tomorrow?

Thanks Matt
________________________________

**Reply by vincentx1908** [reply]
Should be free by 7:30 tonight, is that a good time for you? 
cheers

**Comment by MatthewJRobertson94** [comment]
Should be ok maybe 8 would be easier for me but will have to see

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
